### PR TITLE
Rococo/Wococo bridge hub relay integration

### PR DESCRIPTION
**Pull request by bkontur**

### CODE REVIEW

File: relays/client-bridge-hub-wococo/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+		bp_bridge_hub_wococo::FROM_BRIDGE_HUB_WOCOCO_MESSAGE_DETAILS_METHOD;
+
+	const PAY_INBOUND_DISPATCH_FEE_WEIGHT_AT_CHAIN: Weight =
+		bp_bridge_hub_wococo::PAY_INBOUND_DISPATCH_FEE_WEIGHT;
+	const MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX: MessageNonce =
+		bp_bridge_hub_wococo::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+	const MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX: MessageNonce =
+		bp_bridge_hub_wococo::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+
+	// TODO:check-parameter
```
Comment (svyatonik) [inline relays/client-bridge-hub-wococo/src/lib.rs:136]: Please remove TODO

File: relays/client-bridge-hub-wococo/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+	type WeightToFee = bp_bridge_hub_wococo::WeightToFee;
+	// TODO:check-parameter
```
Comment (svyatonik) [inline relays/client-bridge-hub-wococo/src/lib.rs:138]: Same as with Rococo: That's a good TODO - WeightInfo is used to compute message delivery transaction limits. I think it is fine to use () for initial estimations (relay leaves some reserve anyway && () weights are probably larger than actual weights), but later we'll need to use weights from the parachains. Can you, please, file an issue for that and mention this issue along the TODO? Thanks!

**Review by svyatonik — APPROVED** [review]
All is good, thank you! I've left some comments - feel free to ignore/postpone - they're not critical and we may merge as is

File: primitives/chain-bridge-hub-rococo/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+use bp_runtime::{decl_bridge_finality_runtime_apis, decl_bridge_messages_runtime_apis};
+use frame_support::{
+	parameter_types,
+	sp_runtime::{FixedU128, MultiAddress, MultiSigner},
+	Parameter,
+};
+use sp_std::prelude::*;
+
+pub type BridgeHubRococo = PolkadotLike;
+pub type WeightToFee = frame_support::weights::IdentityFee<Balance>;
```
[the remark this answers is not in the document]
Comment (bkontur) [reply primitives/chain-bridge-hub-rococo/src/lib.rs]: ok, thank you, I aligned it with runtime

**Pull request merged**